packages/desktop/components/ManaBox.svelte

```diff
@@ -71,6 +71,11 @@
                 estimatedManaCost = undefined
                 errorMessage = localize('general.insufficientManaGeneration')
             }
+            if (transactionInfo.preparedTransactionError.message?.includes('cannot be moved off block issuer')) {
```

Comment:
If we want this can be another PR, but the logic behind mana allotments should be fixed so users dont have the impression that they can allot more mana than they actually can

Reply:
opened a separate issue https://github.com/iotaledger/firefly/issues/8535